### PR TITLE
fullheader done

### PR DESCRIPTION
**Pull request by peelerzin**
Header Title + Icon combined together made final header.

### CODE REVIEW

File: src/components/IconButton.tsx [hunk 1]
```diff
@@ -9,7 +9,7 @@ type IconButton = {
 export function IconButton(props: IconButton) {
   return (
     <Link href="/account">
-      <CgProfile className="w-6 h-6 text-primary-default-Solid" />
+      <CgProfile className="w-12 h-12 text-primary-default-Solid" />
```
Comment (NoHara42) [inline src/components/IconButton.tsx:12]: you could pass in a prop to this IconButton component that allows a user of the component to add classes

File: src/components/FullHeader.tsx [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+type FullHeader = {
+  icon: string;
+  name: string;
+};
+
+export function FullHeader(props: FullHeader) {
+  return (
+    <div className="flex items-center gap-24">
+<HeaderTitle name={""}></HeaderTitle>
```
Comment (NoHara42) [inline src/components/FullHeader.tsx:13]: why is name hard coded? couldn't we also pass a prop in here?

**Review by NoHara42 — CHANGES_REQUESTED** [review]
see changes

File: src/components/IconButton.tsx (hunk 1 above)
Comment (peelerzin) [reply src/components/IconButton.tsx:12]: ha sorry, just realized this was pending for a week. i thought it was done and picked others issues in between. Im going to look at what you say.

**Review by NoHara42 — APPROVED** [review]
LGTM

**Pull request merged**